health/health.go

```diff
@@ -1159,14 +1219,10 @@ func (t *Tracker) updateBuiltinWarnablesLocked() {
 		t.setHealthyLocked(derpRegionErrorWarnable)
 	}
 
-	if len(t.controlHealth) > 0 {
```

Comment:
This change made control health message(s) disappear from `tailscale status` and the `tailscaled_health_messages` metric. A bit surprising that we don't have a test for this.

The fix is quite simple:

```
diff --git a/health/health.go b/health/health.go
index 0daf58066..450419c21 100644
--- a/health/health.go
+++ b/health/health.go
@@ -1039,6 +1039,9 @@ func (t *Tracker) stringsLocked() []string {
                        result = append(result, w.Text(ws.Args))
                }
        }
+       for _, m := range t.controlMessages {
+               result = append(result, m.Text)
+       }
        return result
 }
```

Reply:
Fixed and tested in https://github.com/tailscale/tailscale/pull/15839/commits/6082cac99ab0ab3a3a2ccf21ac6bda1aae4597ed and https://github.com/tailscale/tailscale/pull/15839/commits/f6d8926bbd7774ab51324b3b439a0946552e3df6